pages/gallery/_.vue

```diff
@@ -93,8 +89,7 @@
           { hid: 'description', name: 'description', content: this.description },
           { hid: 'og:title', property: 'og:title', content: this.title },
           { hid: 'og:image', property: 'og:image', content: this.shareMediaUrl },
-          { hid: 'og:type', property: 'og:type', content: 'article' },
-          { hid: 'og:url', property: 'og:url', content: this.canonicalURL }
```

Comment:
This page now does not have a og:url. Is that correct?
There should be one, at least, according to https://ogp.me/#metadata .

Reply:
it does, but it doesn't update. This is still in progress